topics/cst/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: 
+display_name: Concrete syntax tree
+released: 
+short_description: A concrete syntax tree is a low level representation of the parsed source in the structure defined by a grammar description.
+topic: cst
+aliases: parse-tree, parsing-tree, derivation-tree
+url:
```

Comment:
I think we can remove these properties since they are empty:
`created_by`, `released`, `url`

Reply:
Ah, right. Did that 👍🏻